### PR TITLE
Refactor for Alps64

### PR DESCRIPTION
**Pull request by noroadsleft**
Refactor `keyboards/alps64/`:

- deletes `LAYOUT_kc` macro from `alps64.h`
- modifies `keymaps/default/keymap.c` to use `LAYOUT_all` macro

### CODE REVIEW

**Comment by noroadsleft** [comment]
A procedural question @drashna (who will probably be the one reviewing this):

Have my PRs today been manageable/well documented? Anything I could do better?

File: keyboards/alps64/alps64.h [hunk 1]
```diff
@@ -106,21 +106,4 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
     { K70,   K71,   K72,   K73, K74, K75, K76,   K77 } \
 }
 
-#define LAYOUT_kc( \
```
Comment (mechmerlin) [inline keyboards/alps64/alps64.h]: I am not in favor of removing the LAYOUT_kc macro as people do find this useful. 

It's fine that we made LAYOUT_all the macro used by the default keymap, but we should keep LAYOUT_kc

**Comment by mechmerlin** [comment]
While you're refactoring this, it would be great to change LAYOUT_standard_60 to LAYOUT_60_ansi

**Comment by mechmerlin** [comment]
You may also want to put a `LAYOUTS = 60_ansi` in rules.mk

**Comment by mechmerlin** [comment]
looks good!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**